md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ext/AnalyticsAppAPI.java

```diff
@@ -0,0 +1,141 @@
+                jpaProperties = Dao2TableUtil.buildJPAProperties("analytics-app", "category", map.get(string), "category Name");
+                propertiesList.add(jpaProperties);
+                LOGGER.debug("appproperties_category" + map.get(string));
+
+            }
+            else if (string.startsWith("appproperties_appname")) {
+                jpaProperties = Dao2TableUtil.buildJPAProperties("analytics-app", "app-name", map.get(string), "Appname");
+                propertiesList.add(jpaProperties);
+                LOGGER.debug("appproperties_appname" + map.get(string));
+            }
+            else if (string.startsWith("appproperties_appdesc")) {
+                jpaProperties = Dao2TableUtil.buildJPAProperties("analytics-app", "app-desc", map.get(string), "appdesc");
+                propertiesList.add(jpaProperties);
+                LOGGER.debug("appproperties_appdesc" + map.get(string));
+            }
+            else if (string.startsWith("appproperties_questionsjson")) {
+                jpaProperties = Dao2TableUtil.buildJPAProperties("analytics-app", "questions-json", map.get(string), "questions json");
+                propertiesList.add(jpaProperties);
+                LOGGER.debug("appproperties_questionsjson" + map.get(string));
+            }
+            else if (string.startsWith("appproperties_dashboardurl")) {
+                jpaProperties = Dao2TableUtil.buildJPAProperties("analytics-app", "dashboard-url", map.get(string), "dashboard url");
+                propertiesList.add(jpaProperties);
+                LOGGER.debug("appproperties_dashboardurl" + map.get(string));
+            }
+            else if (string.startsWith("appproperties_ddpurl")) {
+                jpaProperties = Dao2TableUtil.buildJPAProperties("analytics-app", "ddp-url", map.get(string), "DDP url");
+                propertiesList.add(jpaProperties);
+                LOGGER.debug("appproperties_ddpurl" + map.get(string));
+            }
+            else if (string.startsWith("appImage")) {
+                jpaProperties = Dao2TableUtil.buildJPAProperties("analytics-app", "app-image", map.get(string), "appimage");
+                propertiesList.add(jpaProperties);
+                LOGGER.debug("appproperties_appimage" + map.get(string));
+            }
+
+
+
+        }
+
+
+        com.wipro.ats.bdre.md.dao.jpa.Process parentProcess = Dao2TableUtil.buildJPAProcess(37, processName, processDescription, 1,busDomainId);
+        com.wipro.ats.bdre.md.dao.jpa.Process childProcess1 = Dao2TableUtil.buildJPAProcess(38, "subprocess1 of "+processName, "subprocess1 of "+processDescription, 1,busDomainId);
+    //    com.wipro.ats.bdre.md.dao.jpa.Process childProcess2 = Dao2TableUtil.buildJPAProcess(38, "subprocess2 of "+processName, "subprocess1 of "+processDescription, 1,busDomainId);
+
+        List<Process> childProcesses = new ArrayList<Process>();
+        childProcesses.add(childProcess1);
+   //     childProcesses.add(childProcess2);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)